### DISCUSSION TITLE
Show a notification or have a dynamic subheading

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#9809] [Help]
If there is a GET parameter attached when loading the `List` of a resource, I would like to show a toast/flash or have a special subheading. Is there an easy way to achieve this?

**Comment by leandrocfe** [comment]
You can use [deferLoading](https://filamentphp.com/docs/3.x/tables/advanced#deferring-loading) in the table
```php
$table->deferLoading()
```
and customize the subheading

```php
public function getSubheading(): ?string
{
    return ! $this->getTable()->isLoaded() ? 'Loading...' : parent::getSubheading();
}
```
Is this?
